scripts/rollup/build.js

```diff
@@ -56,8 +56,8 @@ function createBundle({entryPath, bundleType, destName}) {
   ]);
 
   rollup({
-    input: entryPath,
-    plugins: plugins,
+    entryPath,
```

Comment:
It will not work, you should rename `entryPath` variables for input and then use
```rollup({input})```
because input is a needed param for Rollup